### PR TITLE
Fix deprecation warnings and migrate to rubocop-govuk

### PR DESCRIPTION
**Pull request by AlanGabbianelli**
- Fix deprecation warnings
- Migrate to `rubocop-govuk`

See commit messages for more detail.

Trello card: https://trello.com/c/Ik7ulDXQ/1768-3-upgrade-info-frontend-to-rails-6

### CODE REVIEW

**Comment by AlanGabbianelli** [comment]
That's a good point. I'll find out.

File: lib/slimmer/railtie.rb [hunk 1]
```diff
@@ -5,10 +5,20 @@ class Railtie < Rails::Railtie
...
-      app_name = ENV['GOVUK_APP_NAME'] || app.class.parent_name
+      app_name = ENV['GOVUK_APP_NAME'] || module_parent_name
       slimmer_config = slimmer_config.reverse_merge(app_name: app_name)
 
       app.middleware.use Slimmer::App, slimmer_config
     end
+
+  private
+
+    def module_parent_name
```
Comment (issyl0) [inline lib/slimmer/railtie.rb]: We should remember to remove this when all apps are on Rails 6.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+* Fix deprecation warning. (#237)
```
Comment (issyl0) [inline CHANGELOG.md]: Is it worth a further sentence saying what the deprecation warning was?

File: lib/slimmer/railtie.rb (hunk 1 above)
Comment (AlanGabbianelli) [reply lib/slimmer/railtie.rb]: Should I add a `TODO`?

File: CHANGELOG.md (hunk 2 above)
Comment (AlanGabbianelli) [reply CHANGELOG.md]: I don't know. Is it? 😄

File: lib/slimmer/railtie.rb (hunk 1 above)
Comment (issyl0) [reply lib/slimmer/railtie.rb]: No, it's fine.

File: CHANGELOG.md (hunk 2 above)
Comment (AlanGabbianelli) [reply CHANGELOG.md]: Done

File: lib/slimmer/railtie.rb (hunk 1 above)
Comment (AlanGabbianelli) [reply lib/slimmer/railtie.rb]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (issyl0) [reply CHANGELOG.md]: Sorry, I shouldn't phrase things as questions. 😅

File: lib/tasks/lint.rake [hunk 3]
```diff
@@ -1,4 +1,4 @@
-desc "Run govuk-lint with similar params to CI"
```
Comment (issyl0) [inline lib/tasks/lint.rake]: We don't need this file at all. `bundle exec rubocop` is fewer keystrokes than `bundle exec rake lint`. Basically, we want to move away from wrapper scripts and now we can run `rubocop` directly we don't need to wrap things and obscure the parameters. More context is in https://trello.com/c/IEUuFOVa/129-remove-the-lint-rake-task-from-apps-that-have-it (the developer tooling group board).

**Pull request merged**